front/app/api/campaigns/useUpdateCampaign.ts

```diff
@@ -0,0 +1,27 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+import fetcher from 'utils/cl-react-query/fetcher';
+import campaignsKeys from './keys';
+import { ICampaign, IUpdateCampaignProperties } from './types';
+
+const updateCampaign = async ({
+  id: campaignId,
+  campaign,
+}: IUpdateCampaignProperties) =>
+  fetcher<ICampaign>({
+    path: `/campaigns/${campaignId}`,
+    action: 'patch',
+    body: campaign,
```

Comment:
Did you test if this works? Because looking at other, similar hooks, it looks like you need to wrap this:

`body: { campaign },
`

Reply:
it works both ways, but the way you propose is how it was before so changing to that